go/libraries/doltcore/sqle/database_provider.go

```diff
@@ -128,6 +130,9 @@ func (p DoltDatabaseProvider) CreateDatabase(ctx *sql.Context, name string) erro
 	p.mu.Lock()
 	defer p.mu.Unlock()
 
+	// Throw warning to users that this is a temporary database that does not persist after the session exits.
+	ctx.Warn(createDbWC, "CREATE DATABASE creates an inmemory database that does not persist after the server exits.")
```

Comment:
in memory

Add:

Dolt currently only supports a single disk backed database created by dolt init.